commit 5860b35ff6c7241d1c35a1b3197b45e5c9ff86cf
Author: fanquake <fanquake@gmail.com>
Date:   Thu Jun 29 11:52:43 2023 +0100

    ld64: add support for -no_fixup_chains
    
    This is added in later versions, and is required if we want to be able
    to disable fixup_chains, for use in security tests.

diff --git a/cctools/ld64/src/ld/Options.cpp b/cctools/ld64/src/ld/Options.cpp
index 15e8e88..b6580af 100644
--- a/cctools/ld64/src/ld/Options.cpp
+++ b/cctools/ld64/src/ld/Options.cpp
@@ -4128,6 +4128,9 @@ void Options::parse(int argc, const char* argv[])
 			else if ( strcmp(arg, "-fixup_chains") == 0 ) {
 				fMakeChainedFixups = true;
 			}
+			else if ( strcmp(arg, "-no_fixup_chains") == 0 ) {
+				fMakeChainedFixups = false;
+			}
 			else if (strcmp(arg, "-debug_variant") == 0) {
 			    fDebugVariant = true;
             }
